.github/workflows/test.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         bot: ['base']
-        lang: ['eng', 'afr']
+        lang: ['eng', 'afr', 'xho']
```

Comment:
not even sure if this is really needed

Reply:
Possibly not, but we do a `rasa data validate` on the domain file, so probably good to have that for each domain file to validate that it's correct